apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -215,14 +214,14 @@ const tokenTransactionInvalid = [
 ];
 
 test.describe("Send flows", () => {
-  test.beforeAll(async () => {
-    process.env.DISABLE_TRANSACTION_BROADCAST =
-      new Date().getDay() === 1 && isRunningInScheduledWorkflow() ? "0" : "1";
-  });
   //Warning 🚨: Test may fail due to the GetAppAndVersion issue - Jira: LIVE-12581 or insufficient funds
 
   for (const transaction of transactionE2E) {
     test.describe("Send from 1 account to another", () => {
+      test.beforeAll(async () => {
+        process.env.DISABLE_TRANSACTION_BROADCAST = "0";
```

Comment:
We are enabling Send tests broadcast for all executions ?

Reply:
Leftover from testing activities, removed